### ISSUE TITLE
Make certificatePolicies:policyQualifiers required in the profile

### ISSUE THREAD

**Issue by lachellel** [uspki/policies#511]
Suggestion to make the certificatePolicies:policyQualifier required instead of optional

- [Profile Link](https://github.com/uspki/policies/blob/master/certificate-profile-server-authentication.md)
- certificatePolicies:policyQualifiers 
- policyQualifierId id-qt 1 
- qualifier:cPSuri

@ryancdickson @konklone 

Thoughts?

**Comment by techliaison** [comment]
certificatePolicies is already mandatory.
I believe you are asking about making the policy Qualifier of URI to the CPS mandatory - in my opinion this just inflates the size of the certificates unnecessarily.  But since we are mandating public publication of the CPS, I have no objection.

**Comment by lachellel** [comment]
True - I should have been more clear in the issue title.  I was writing it in parallel with reviewing the cert samples.  I'll update the title to include policyQualifier.

**Comment by ryancdickson** [comment]
- Samples below to include the requested update. 
- Temporarily pointing to http://repository.pki-lab.gov/cps - open to changing, though.

Any opinions? @lachellel @konklone @ChristopherRC

***DV***
```
-----BEGIN CERTIFICATE-----
MIIGUDCCBTigAwIBAgIVAIHf+rlkw4iJ6j6RQ2Uijsob/L+MMA0GCSqGSIb3DQEB
CwUAMEwxCzAJBgNVBAYTAlVTMRgwFgYDVQQKEw9VLlMuIEdvdmVybm1lbnQxIzAh
BgNVBAMTGlVTIEZlZGVyYWwgVGVzdCBUTFMgQ0EgMTAwMB4XDTE4MDQwNDIwNDcz
OVoXDTE5MDUwNDIwNDczOVowLzELMAkGA1UEBhMCVVMxIDAeBgNVBAMTF2RlbG9y
ZWFuLWR2LnBraS1sYWIuZ292MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKC
AQEAmnfxWcm9wb9BrfAlsExmiofx4dvpuo4KDlujvfMdzmXFBVylALDGZG35R9ua
hL/E+J1wyK3n0cdiSVkA18603Sh7fmXpgEk6Rmj+3tiUvVCnA4ogZgb5G3uasSKq
2odZJoSwx/V+EGsx3nHMfmo1pjKSlue12fpGzFfrApKU2DoRnjIOWeuj1RXkBjQs
Dx7uilDRXrDhZVyNP5p43UN263gSMZi8VK4veWokFwKjKZWNTQHkw2M7aXm02/lp
51kvkc2W6m+FcAjcYCWrIhMpe2bemH+HnPOOMvIZnR34cNPOgtCT31D3e4bpjDuh
lWk8AV++nUHdZo2Y6Z3qPmOujQIDAQABo4IDRDCCA0AwUgYDVR0gBEswSTA9Bgpg
hkgBZQMCAQMrMC8wLQYIKwYBBQUHAgEWIWh0dHA6Ly9yZXBvc2l0b3J5LnBraS1s
YWIuZ292L2NwczAIBgZngQwBAgEwHwYDVR0jBBgwFoAUfk4r8WSyrat/YpEwMNVU
xtI7VeEwHQYDVR0OBBYEFPSbDuGdvQnyNg3Xg/MDGXn2mmV3MAwGA1UdEwEB/wQC
MAAwDgYDVR0PAQH/BAQDAgWgMB0GA1UdJQQWMBQGCCsGAQUFBwMBBggrBgEFBQcD
AjAiBgNVHREEGzAZghdkZWxvcmVhbi1kdi5wa2ktbGFiLmdvdjB9BggrBgEFBQcB
AQRxMG8wKAYIKwYBBQUHMAGGHGh0dHA6Ly9vY3NwLmRpc2EucGtpLWxhYi5nb3Yw
QwYIKwYBBQUHMAKGN2h0dHA6Ly9jZXJ0Lmlzc3VpbmcucGtpLWxhYi5nb3Yvc2ln
bi9VU0ZFRFRFU1RDQTEwMC5wN2MwRwYDVR0fBEAwPjA8oDqgOIY2aHR0cDovL2lz
c3VpbmcuZGlzYS5wa2ktbGFiLmdvdi9jcmwvVVNGRURURVNUQ0ExMDAuY3JsMIIB
fwYKKwYBBAHWeQIEAgSCAW8EggFrAWkAdgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAWHDUvAbAAAEAwBHAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
dwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHDUvEhAAAEAwBI
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHYAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAFhw1LwxQAABAMARwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
MA0GCSqGSIb3DQEBCwUAA4IBAQBH561yMbnmtpiHxWGZxPzHeXC37zozAwH9GUk/
QagqdviwaRCwsIp6rbZGmomJtCZwXo+7hepxu0QSO2eyoWHCZUNgEYYHJfEfDGfj
Buzh6W3Q7djHPWVQGcIE05dzO2Xje7pQFn7a7GP6KKQ1gtEwquQfzYjq4XlWoaMN
wAhdGYboHULfW7SQlHZHJj6yKVAJxVMdqviiH09aD+jjhsYyaDQyaJxSUU95ZNEY
inSwcwxUReaIO97E1Ajbq4mTs7gx/y4Q17Z+7+TT0ML7mOMVQyPBlBbXaKOB7R6O
JDwYkPp97+Ln3uaiS0Ds+R6lD6IBuEtMKtym9/wBWs08vOTh
-----END CERTIFICATE-----
```
```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            81:df:fa:b9:64:c3:88:89:ea:3e:91:43:65:22:8e:ca:1b:fc:bf:8c
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, O=U.S. Government, CN=US Federal Test TLS CA 100
        Validity
            Not Before: Apr  4 20:47:39 2018 GMT
            Not After : May  4 20:47:39 2019 GMT
        Subject: C=US, CN=delorean-dv.pki-lab.gov
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
                    00:9a:77:f1:59:c9:bd:c1:bf:41:ad:f0:25:b0:4c:
                    66:8a:87:f1:e1:db:e9:ba:8e:0a:0e:5b:a3:bd:f3:
                    1d:ce:65:c5:05:5c:a5:00:b0:c6:64:6d:f9:47:db:
                    9a:84:bf:c4:f8:9d:70:c8:ad:e7:d1:c7:62:49:59:
                    00:d7:ce:b4:dd:28:7b:7e:65:e9:80:49:3a:46:68:
                    fe:de:d8:94:bd:50:a7:03:8a:20:66:06:f9:1b:7b:
                    9a:b1:22:aa:da:87:59:26:84:b0:c7:f5:7e:10:6b:
                    31:de:71:cc:7e:6a:35:a6:32:92:96:e7:b5:d9:fa:
                    46:cc:57:eb:02:92:94:d8:3a:11:9e:32:0e:59:eb:
                    a3:d5:15:e4:06:34:2c:0f:1e:ee:8a:50:d1:5e:b0:
                    e1:65:5c:8d:3f:9a:78:dd:43:76:eb:78:12:31:98:
                    bc:54:ae:2f:79:6a:24:17:02:a3:29:95:8d:4d:01:
                    e4:c3:63:3b:69:79:b4:db:f9:69:e7:59:2f:91:cd:
                    96:ea:6f:85:70:08:dc:60:25:ab:22:13:29:7b:66:
                    de:98:7f:87:9c:f3:8e:32:f2:19:9d:1d:f8:70:d3:
                    ce:82:d0:93:df:50:f7:7b:86:e9:8c:3b:a1:95:69:
                    3c:01:5f:be:9d:41:dd:66:8d:98:e9:9d:ea:3e:63:
                    ae:8d
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Certificate Policies:
                Policy: 2.16.840.1.101.3.2.1.3.43
                  CPS: http://repository.pki-lab.gov/cps
                Policy: 2.23.140.1.2.1

            X509v3 Authority Key Identifier:
                keyid:7E:4E:2B:F1:64:B2:AD:AB:7F:62:91:30:30:D5:54:C6:D2:3B:55:E1

            X509v3 Subject Key Identifier:
                F4:9B:0E:E1:9D:BD:09:F2:36:0D:D7:83:F3:03:19:79:F6:9A:65:77
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage:
                TLS Web Server Authentication, TLS Web Client Authentication
            X509v3 Subject Alternative Name:
                DNS:delorean-dv.pki-lab.gov
            Authority Information Access:
                OCSP - URI:http://ocsp.disa.pki-lab.gov
                CA Issuers - URI:http://cert.issuing.pki-lab.gov/sign/USFEDTESTCA100.p7c

            X509v3 CRL Distribution Points:

                Full Name:
                  URI:http://issuing.disa.pki-lab.gov/crl/USFEDTESTCA100.crl

            CT Precertificate SCTs:
                Signed Certificate Timestamp:
                    Version   : v1(0)
                    Log ID    : 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00
                    Timestamp : Feb 23 15:40:48.027 2018 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00
                Signed Certificate Timestamp:
                    Version   : v1(0)
                    Log ID    : 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00
                    Timestamp : Feb 23 15:40:48.289 2018 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00
                Signed Certificate Timestamp:
                    Version   : v1(0)
                    Log ID    : 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00
                    Timestamp : Feb 23 15:40:48.197 2018 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00
    Signature Algorithm: sha256WithRSAEncryption
         47:e7:ad:72:31:b9:e6:b6:98:87:c5:61:99:c4:fc:c7:79:70:
         b7:ef:3a:33:03:01:fd:19:49:3f:41:a8:2a:76:f8:b0:69:10:
         b0:b0:8a:7a:ad:b6:46:9a:89:89:b4:26:70:5e:8f:bb:85:ea:
         71:bb:44:12:3b:67:b2:a1:61:c2:65:43:60:11:86:07:25:f1:
         1f:0c:67:e3:06:ec:e1:e9:6d:d0:ed:d8:c7:3d:65:50:19:c2:
         04:d3:97:73:3b:65:e3:7b:ba:50:16:7e:da:ec:63:fa:28:a4:
         35:82:d1:30:aa:e4:1f:cd:88:ea:e1:79:56:a1:a3:0d:c0:08:
         5d:19:86:e8:1d:42:df:5b:b4:90:94:76:47:26:3e:b2:29:50:
         09:c5:53:1d:aa:f8:a2:1f:4f:5a:0f:e8:e3:86:c6:32:68:34:
         32:68:9c:52:51:4f:79:64:d1:18:8a:74:b0:73:0c:54:45:e6:
         88:3b:de:c4:d4:08:db:ab:89:93:b3:b8:31:ff:2e:10:d7:b6:
         7e:ef:e4:d3:d0:c2:fb:98:e3:15:43:23:c1:94:16:d7:68:a3:
         81:ed:1e:8e:24:3c:18:90:fa:7d:ef:e2:e7:de:e6:a2:4b:40:
         ec:f9:1e:a5:0f:a2:01:b8:4b:4c:2a:dc:a6:f7:fc:01:5a:cd:
         3c:bc:e4:e1
```

***OV***
```
-----BEGIN CERTIFICATE-----
MIIGiDCCBXCgAwIBAgIUaNXPvv8wuoyna/e6+qnCJ+7woWwwDQYJKoZIhvcNAQEL
BQAwTDELMAkGA1UEBhMCVVMxGDAWBgNVBAoTD1UuUy4gR292ZXJubWVudDEjMCEG
A1UEAxMaVVMgRmVkZXJhbCBUZXN0IFRMUyBDQSAxMDAwHhcNMTgwNDA0MjA0NzAy
WhcNMTkwNTA0MjA0NzAyWjBoMQswCQYDVQQGEwJVUzEYMBYGA1UEChMPVS5TLiBH
b3Zlcm5tZW50MR0wGwYDVQQIExREaXN0cmljdCBvZiBDb2x1bWJpYTEgMB4GA1UE
AxMXZGVsb3JlYW4tb3YucGtpLWxhYi5nb3YwggEiMA0GCSqGSIb3DQEBAQUAA4IB
DwAwggEKAoIBAQCad/FZyb3Bv0Gt8CWwTGaKh/Hh2+m6jgoOW6O98x3OZcUFXKUA
sMZkbflH25qEv8T4nXDIrefRx2JJWQDXzrTdKHt+ZemASTpGaP7e2JS9UKcDiiBm
Bvkbe5qxIqrah1kmhLDH9X4QazHeccx+ajWmMpKW57XZ+kbMV+sCkpTYOhGeMg5Z
66PVFeQGNCwPHu6KUNFesOFlXI0/mnjdQ3breBIxmLxUri95aiQXAqMplY1NAeTD
YztpebTb+WnnWS+RzZbqb4VwCNxgJasiEyl7Zt6Yf4ec844y8hmdHfhw086C0JPf
UPd7humMO6GVaTwBX76dQd1mjZjpneo+Y66NAgMBAAGjggNEMIIDQDBSBgNVHSAE
SzBJMD0GCmCGSAFlAwIBAywwLzAtBggrBgEFBQcCARYhaHR0cDovL3JlcG9zaXRv
cnkucGtpLWxhYi5nb3YvY3BzMAgGBmeBDAECAjAfBgNVHSMEGDAWgBR+TivxZLKt
q39ikTAw1VTG0jtV4TAdBgNVHQ4EFgQU9JsO4Z29CfI2DdeD8wMZefaaZXcwDAYD
VR0TAQH/BAIwADAOBgNVHQ8BAf8EBAMCBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEG
CCsGAQUFBwMCMCIGA1UdEQQbMBmCF2RlbG9yZWFuLW92LnBraS1sYWIuZ292MH0G
CCsGAQUFBwEBBHEwbzAoBggrBgEFBQcwAYYcaHR0cDovL29jc3AuZGlzYS5wa2kt
bGFiLmdvdjBDBggrBgEFBQcwAoY3aHR0cDovL2NlcnQuaXNzdWluZy5wa2ktbGFi
Lmdvdi9zaWduL1VTRkVEVEVTVENBMTAwLnA3YzBHBgNVHR8EQDA+MDygOqA4hjZo
dHRwOi8vaXNzdWluZy5kaXNhLnBraS1sYWIuZ292L2NybC9VU0ZFRFRFU1RDQTEw
MC5jcmwwggF/BgorBgEEAdZ5AgQCBIIBbwSCAWsBaQB2AAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAABYcNS8BsAAAQDAEcAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAB3AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABYcNS
8SEAAAQDAEgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAdgAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHDUvDFAAAEAwBHAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAwDQYJKoZIhvcNAQELBQADggEBAEZBwl+igSB72zuLEweMEOcouR0a
pxLw7iECajWexvuXcgbnwjWwIhs7cWPSg1QqyvgZrQhzPNIq2O7kJliswXXYIpzA
r6pgOVzeA47fs/XNqfs9e7JoNKoc6sruCHjKYqGscT636BtZuwK+25riSEvDbjdH
wNH1v8ZnmBvtgYVcIxQvpJaK+W43W+NwlobIcHq99wh5LoxX5KkZyQxcdmpaNqfI
d4Fkz3nt3fNjME2b0n6dv9CMId7us9u4NbBLwK8Uvaaj4Py/l2mPqtG46dpLKTCe
MAdj6eipZxHo9dcIDY0G6JIzZ27/S/b83fAoPeXYFJLbLdhnJiH7TDPkjZU=
-----END CERTIFICATE-----
```
```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            68:d5:cf:be:ff:30:ba:8c:a7:6b:f7:ba:fa:a9:c2:27:ee:f0:a1:6c
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, O=U.S. Government, CN=US Federal Test TLS CA 100
        Validity
            Not Before: Apr  4 20:47:02 2018 GMT
            Not After : May  4 20:47:02 2019 GMT
        Subject: C=US, O=U.S. Government, ST=District of Columbia, CN=delorean-ov.pki-lab.gov
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
                    00:9a:77:f1:59:c9:bd:c1:bf:41:ad:f0:25:b0:4c:
                    66:8a:87:f1:e1:db:e9:ba:8e:0a:0e:5b:a3:bd:f3:
                    1d:ce:65:c5:05:5c:a5:00:b0:c6:64:6d:f9:47:db:
                    9a:84:bf:c4:f8:9d:70:c8:ad:e7:d1:c7:62:49:59:
                    00:d7:ce:b4:dd:28:7b:7e:65:e9:80:49:3a:46:68:
                    fe:de:d8:94:bd:50:a7:03:8a:20:66:06:f9:1b:7b:
                    9a:b1:22:aa:da:87:59:26:84:b0:c7:f5:7e:10:6b:
                    31:de:71:cc:7e:6a:35:a6:32:92:96:e7:b5:d9:fa:
                    46:cc:57:eb:02:92:94:d8:3a:11:9e:32:0e:59:eb:
                    a3:d5:15:e4:06:34:2c:0f:1e:ee:8a:50:d1:5e:b0:
                    e1:65:5c:8d:3f:9a:78:dd:43:76:eb:78:12:31:98:
                    bc:54:ae:2f:79:6a:24:17:02:a3:29:95:8d:4d:01:
                    e4:c3:63:3b:69:79:b4:db:f9:69:e7:59:2f:91:cd:
                    96:ea:6f:85:70:08:dc:60:25:ab:22:13:29:7b:66:
                    de:98:7f:87:9c:f3:8e:32:f2:19:9d:1d:f8:70:d3:
                    ce:82:d0:93:df:50:f7:7b:86:e9:8c:3b:a1:95:69:
                    3c:01:5f:be:9d:41:dd:66:8d:98:e9:9d:ea:3e:63:
                    ae:8d
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Certificate Policies:
                Policy: 2.16.840.1.101.3.2.1.3.44
                  CPS: http://repository.pki-lab.gov/cps
                Policy: 2.23.140.1.2.2

            X509v3 Authority Key Identifier:
                keyid:7E:4E:2B:F1:64:B2:AD:AB:7F:62:91:30:30:D5:54:C6:D2:3B:55:E1

            X509v3 Subject Key Identifier:
                F4:9B:0E:E1:9D:BD:09:F2:36:0D:D7:83:F3:03:19:79:F6:9A:65:77
            X509v3 Basic Constraints: critical
                CA:FALSE
            X509v3 Key Usage: critical
                Digital Signature, Key Encipherment
            X509v3 Extended Key Usage:
                TLS Web Server Authentication, TLS Web Client Authentication
            X509v3 Subject Alternative Name:
                DNS:delorean-ov.pki-lab.gov
            Authority Information Access:
                OCSP - URI:http://ocsp.disa.pki-lab.gov
                CA Issuers - URI:http://cert.issuing.pki-lab.gov/sign/USFEDTESTCA100.p7c

            X509v3 CRL Distribution Points:

                Full Name:
                  URI:http://issuing.disa.pki-lab.gov/crl/USFEDTESTCA100.crl

            CT Precertificate SCTs:
                Signed Certificate Timestamp:
                    Version   : v1(0)
                    Log ID    : 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00
                    Timestamp : Feb 23 15:40:48.027 2018 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00
                Signed Certificate Timestamp:
                    Version   : v1(0)
                    Log ID    : 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00
                    Timestamp : Feb 23 15:40:48.289 2018 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00
                Signed Certificate Timestamp:
                    Version   : v1(0)
                    Log ID    : 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00
                    Timestamp : Feb 23 15:40:48.197 2018 GMT
                    Extensions: none
                    Signature : ecdsa-with-SHA256
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:
                                00:00:00:00:00:00:00
    Signature Algorithm: sha256WithRSAEncryption
         46:41:c2:5f:a2:81:20:7b:db:3b:8b:13:07:8c:10:e7:28:b9:
         1d:1a:a7:12:f0:ee:21:02:6a:35:9e:c6:fb:97:72:06:e7:c2:
         35:b0:22:1b:3b:71:63:d2:83:54:2a:ca:f8:19:ad:08:73:3c:
         d2:2a:d8:ee:e4:26:58:ac:c1:75:d8:22:9c:c0:af:aa:60:39:
         5c:de:03:8e:df:b3:f5:cd:a9:fb:3d:7b:b2:68:34:aa:1c:ea:
         ca:ee:08:78:ca:62:a1:ac:71:3e:b7:e8:1b:59:bb:02:be:db:
         9a:e2:48:4b:c3:6e:37:47:c0:d1:f5:bf:c6:67:98:1b:ed:81:
         85:5c:23:14:2f:a4:96:8a:f9:6e:37:5b:e3:70:96:86:c8:70:
         7a:bd:f7:08:79:2e:8c:57:e4:a9:19:c9:0c:5c:76:6a:5a:36:
         a7:c8:77:81:64:cf:79:ed:dd:f3:63:30:4d:9b:d2:7e:9d:bf:
         d0:8c:21:de:ee:b3:db:b8:35:b0:4b:c0:af:14:bd:a6:a3:e0:
         fc:bf:97:69:8f:aa:d1:b8:e9:da:4b:29:30:9e:30:07:63:e9:
         e8:a9:67:11:e8:f5:d7:08:0d:8d:06:e8:92:33:67:6e:ff:4b:
         f6:fc:dd:f0:28:3d:e5:d8:14:92:db:2d:d8:67:26:21:fb:4c:
         33:e4:8d:95
```

Note: Updated slightly [twice] after noticing a typo in my openssl.cnf file. Sorry!

**Comment by lachellel** [comment]
This is a good practice but not required. 
No fix for v1.0 final drafting.
